### PR TITLE
Update fskit package

### PR DESCRIPTION
**Pull request by JL2210**
Update fskit package for compatibility with attr 2.4.48

Tested on:
- [x] x86_64

### CODE REVIEW

File: packages/fskit.rb [hunk 1]
```diff
@@ -3,38 +3,36 @@
...
 
   depends_on 'attr'
 
   def self.patch
-    # for x86_64 architecture, build library should be modified to #{CREW_PREFIX}/lib64
-    if ARCH == "x86_64"
-      puts "Patch build file for x86_64 ..."
-      `sed -i -- 's/lib/lib64/g' buildconf.mk` # replace lib by lib64
-    end
+    system "wget", "https://github.com/JL2210/patches/raw/master/fskit-dd9a8-xattr.patch"
```
Comment (uberhacker) [inline packages/fskit.rb]: Should we add a sha256 checksum on the patch?

Comment (JL2210) [reply packages/fskit.rb]: No. Right now, I'm planning on improving it. It's kind of crude right now. It just totally removes the header file and replaces it with two others.

Comment (uberhacker) [reply packages/fskit.rb]: That doesn't give me much confidence.  We should probably add the sha256 checksum just in case.

Comment (JL2210) [reply packages/fskit.rb]: For now, I'll use a hashed version and add a sha256sum.

Comment (JL2210) [reply packages/fskit.rb]: BTW, patch fails with something along the lines of `patch: **** Only garbage was found in the patch input.` if the patch is corrupted.

Comment (uberhacker) [reply packages/fskit.rb]: Well, with a sha256 checksum, at least they would get a message that tells them to try again.  I'm afraid they may just think the patch is corrupt if it doesn't download cleanly.

Comment (uberhacker) [reply packages/fskit.rb]: Anyway, with binaries we don't have to worry about any of these edge cases.

Comment (JL2210) [reply packages/fskit.rb]: Done.

**Pull request merged**